core/src/test/java/org/infinispan/distribution/BaseDistSyncL1Test.java

```diff
@@ -313,9 +314,9 @@ public void testRemoteGetArrivesButWriteOccursBeforeRegistration() throws Throwa
       // interceptor
       CyclicBarrier getBarrier = new CyclicBarrier(3);
       ownerCache.getAdvancedCache().addInterceptorAfter(new BlockingInterceptor(getBarrier, GetCacheEntryCommand.class, true, false),
-                                                        getL1InterceptorClass());
+              getL1InterceptorClass());
       backupOwnerCache.getAdvancedCache().addInterceptorAfter(new BlockingInterceptor(getBarrier, GetCacheEntryCommand.class, true, false),
-                                                       getL1InterceptorClass());
+              getL1InterceptorClass());
```

Comment:
Yeah the code style import for IntelliJ has continuation indent at 8.  Should we change this?